### DISCUSSION TITLE
Unable to get the list of checkboxes that are checked in form

### DISCUSSION THREAD

**Discussion by shardul-amte** [fastapi/fastapi#6921] [Questions] [answered]
Hello,

I'm unable to get the checked checkboxes in form via Form object in Fast API. Also I could not find anything in documentation for getting data of checkboxes that are checked. Please help ASAP.

Obviously there's work-around for this but this functionality does not seem to available with FastAPI. Awaiting your response. 

Thanks,
Shardul

**Comment by ycd** [comment]
Have you tried looking inside the[ `Request`](https://fastapi.tiangolo.com/advanced/using-request-directly/) object? Edit the question by following the issue template like, what have you tried? Which things didn't work the way you expect etc.

**Comment by shardul-amte** [comment, marked as answer]
Oops my bad. Did not restart the server. The Form object works just fine. Thanks anyways. Will close this issue:)

**Comment by tiangolo** [comment]
Thanks for the help here @ycd ! :clap: :bow: 

Thanks for reporting back and closing the issue @shardul-amte  :+1:

**Comment by aaronted009** [comment]
Related [discussion](https://github.com/tiangolo/fastapi/discussions/11640) with a code example.
